### ISSUE TITLE
[BUG] UI issues due to partial application of (dark) GTK theme

### ISSUE THREAD

**Issue by Ghostbird** [PCGen/pcgen#6422]
### TL;DR; Workaround ###
Parts of the UI not visible when running on GTK based linux? Override the theme in the PCGen startup so it runs with a light theme that works well with PCGen. You can put `export GTK_THEME='Adwaita'` near the top of the `pcgen.sh` file to set it every time the program runs.

### Describe the bug ###
GTK theming is only partially supported, which makes some parts of the UI hard to read in e.g. the dark variant of Cinnamon's standard theme.

### To Reproduce ###
Start PCGen on a system that is running a GTK theme that has colours that wildly differ from the PCGen defaults, such as _Adwaita-dark_.

### Expected behavior ###
All theme colours are applied, and the colours work well together.

### Screenshots ###

**The result of running with a dark theme**
![The result of running with a dark theme](https://user-images.githubusercontent.com/1202149/99193275-e57d9d80-276f-11eb-9a09-6f2e0e654302.png)

**Here the dark theme has been overridden by setting `GTK_THEME=Adwaita` in the shell before running PCGen**
![Here the dark theme has been overridden by setting `GTK_THEME=Adwaita` in the shell before running PCGen](https://user-images.githubusercontent.com/1202149/99193292-f75f4080-276f-11eb-80d0-12a63f38de1a.png)

### Desktop ###
 - OS: Debian 10 with Cinnamon, _Default_ window border theme and _Adwaita-dark_ control theme.
 - Browser: Irrelevant, this is about the desktop application
 - Version v6.09.05

### Character File ###
Irrelevant

### Additional context ###
This is not a big issue, but it would be nice if there was a way to inform the user of this workaround. Ideally apply it automatically until such a time where a dark theme is fully supported.

**Comment by grimreaper** [comment]
We are migrating from Swing to JavaFX. I believe only Swing supports GTK theming. I don't think it is like;y we'll support this.

**Comment by Ghostbird** [comment]
> We are migrating from Swing to JavaFX. I believe only Swing supports GTK theming. I don't think it is like;y we'll support this.

Then I assume that once the migration is complete, it probably won't use the dark theme at all. That would be fine. This issue arose because only some parts of the UI were dark.

**Comment by Ghostbird** [comment]
I'm closing this as it's probably not going to be an issue once the migration is completed. Even if we'd lose dark theme support, the UI would at least work properly.
